uploader.json

```diff
@@ -4883,21 +4883,21 @@
             "docker_container": [
                 {
                     "archive_file_name": "/insights_data/container/rootfs/{EXPANDED_FILE_NAME}",
-                    "file": "{CONTAINER_MOUNT_POINT}/etc/krb5.conf.d/()*.*\\.conf",
+                    "file": "{CONTAINER_MOUNT_POINT}/etc/krb5.conf.d/()*.*",
```

Comment:
@jsvob please remove this change and all changed below it.  This section of the JSON is not currently used for collection.